<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketServer</class>
 <widget class="QWidget" name="SocketServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>520</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>281</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Send Message</string>
   </property>
   <widget class="QTextEdit" name="send_Edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>101</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Receive Message</string>
   </property>
   <widget class="QTextEdit" name="receive_edit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="port_Edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>38</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>41</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prot : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>31</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>441</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>91</y>
     <width>181</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Clients</string>
   </property>
   <widget class="QListView" name="listView_ipPort">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>171</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="ip_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>37</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="listen_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>31</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
